packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillContentStyles.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillContentStylesProps } from '../../../../components/Pill/PillContent';
+import { PillVariables } from './pillVariables';
+
+export const pillContentStyles: ComponentSlotStylesPrepared<PillContentStylesProps, PillVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    fontSize: v.contentFontSize,
+    padding: `${v.contentVerticalPadding} ${v.contentHorizontalPadding}`,
```

Comment:
```suggestion
    padding: `${v.contentPadding}`,
```

Can we use just `content*Padding` in this and other cases?